smac/initial_design/random_configuration_design.py

```diff
@@ -0,0 +1,96 @@
+import sys
+
+import numpy as np
+
+from smac.initial_design.initial_design import InitialDesign
+from smac.tae.execute_ta_run import ExecuteTARun
+from smac.stats.stats import Stats
+from smac.utils.io.traj_logging import TrajLogger
+from smac.scenario.scenario import Scenario
+from smac.tae.execute_ta_run import StatusType
+from smac.runhistory.runhistory import RunHistory
+from smac.utils import constants
+
+__author__ = "Katharina Eggensperger"
+__copyright__ = "Copyright 2016, ML4AAD"
+__license__ = "3-clause BSD"
+
+
+class RandomConfiguration(InitialDesign):
```

Comment:
This class is mostly a copy of the `DefaultConfiguration`, I think this should be generalized.